### PR TITLE
Fix document for Marionette.getOption

### PR DESCRIPTION
**Pull request by 38elements**
This fixes document for `Marionette.getOption()`.

### CODE REVIEW

**Comment by 38elements** [comment]
@denar90 
Would you tell me whether or not the following example is correct?

```javascript
var Obj = Mn.Object.extend({
  foo: 1,
  bar: 2,
  initialize: function() {
    console.log(this.getOption('bar'));
  }
});

var obj = new Obj({
  bar: 3,
  baz: 4
}); // => 3

console.log(Mn.getOption(obj, 'foo')); // => 1
console.log(Mn.getOption(obj, 'bar')); // => 3
console.log(Mn.getOption(obj, 'baz')); // => 4
```

**Comment by 38elements** [comment]
@denar90 
Thank you for your reply.
I made an example.

**Comment by denar90** [comment]
@38elements thanks for adding changes 👍 
I also added one live example.

File: docs/marionette.functions.md [hunk 1]
```diff
@@ -124,16 +126,35 @@ var Obj = Mn.Object.extend({
 
   initialize: function(attributes, options){
```
Comment (paulfalgout) [inline docs/marionette.functions.md]: actually this was incorrect to begin with.  `attributes` isn't a thing.  That'd be true for a `Backbone.Model` but not for an object...

File: docs/marionette.functions.md [hunk 2]
```diff
@@ -124,16 +126,35 @@ var Obj = Mn.Object.extend({
...
   }
 });
 
-var obj1 = new Obj(); // => "bar"
+var obj1 = new Obj(); // => 'bar'
+var obj2 = new Obj({}, { foo: 'quux' }); // => 'quux'
+
+```
+
+[Live example](https://jsfiddle.net/marionettejs/271nn9ht/)
```
Comment (paulfalgout) [inline docs/marionette.functions.md]: this example is forcing the code to work by adding `this.options = options` but that's mostly breaking Marionette.

It is however necessary in a Backbone.Model

File: docs/marionette.functions.md (hunk 1 above)
Comment (38elements) [inline docs/marionette.functions.md]: I am sorry.
I sent pull request #3238 .

**Pull request merged**